app/models/menu.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Menu < ApplicationRecord
+  has_many :cart_items
+
+  enum category: %i[First Main Drinks]
+  enum day: %i[Monday Tuesday Wednesday Thursday Friday]
+
+  validates :title, presence: true
+  validates :price, presence: true
+
+  def title_with_price
+    "#{title} $#{price.to_d}"
```

Comment:
1. Read https://en.wikipedia.org/wiki/Decorator_pattern
2. Read and use https://github.com/drapergem/draper